config/environment.js

```diff
@@ -10,7 +10,7 @@ function getSentryServer(dsn, withProtocol = true) {
 
 module.exports = function(environment) {
   let ENV = {
-    appName                  : process.env.APP_NAME || 'Open Event',
+    appName                  : process.env.APP_NAME || 'eventyay',
```

Comment:
Why are you going in completely opposite direction and making it even worse? Read the repository name. eventyay is an instance. Open Event is the project. I don't understand how to make it more clear

Reply:
https://github.com/fossasia/open-event-frontend/pull/3626#discussion_r347128874

Reply:
> 
> 
> Why are you going in completely opposite direction and making it even worse? Read the repository name. eventyay is an instance. Open Event is the project. I don't understand how to make it more clear

@iamareebjamal  So you are saying that open event is written there intentionally and you want me to change existing eventyay to config.appname i.e. 'Open Event'
![Screenshot (70)_LI](https://user-images.githubusercontent.com/43860289/69363086-c6ac0e80-0cb5-11ea-99bc-52c9080c815d.jpg)

Reply:
Absolutely!